### PR TITLE
add MobileOneBlock and SIoULoss

### PR DESCRIPTION
**Pull request by TUDelftHao**

### CODE REVIEW

**Comment by jerrywgz** [comment]
Thanks for your contribution! Did you train the models using mobileone or siou?

**Comment by TUDelftHao** [comment]
> 

Thank you for your reply. I have verified the functionality of siou and MobileOne block based on the picodet structure. The siou showed an improvement on my custom dataset and work comparable well on the mini-coco(siou: 0.195, giou 0.198). Since I implement siou before the recent releasement of yolov6, which also applies siou as a loss function, there is a little difference between these two versions. Now I am comparing my paddle version with the torch implementation. Updating will be given later if necessary. As for MobileOne block, I just implement the basic block unit instead of the whole backbone. The way I verify it is to replace this block (with k=1) with the normal convolution block in the head of picodet, which gives an increase of inference speed by 11.4% in CPU mode with comparable mAP performance on my custom dataset.

**Comment by TUDelftHao** [comment]
> Thanks for your contribution! Did you train the models using mobileone or siou?

Thank you for your reply. I have verified the functionality of siou and MobileOne block based on the picodet structure. The siou showed an improvement on my custom dataset and work comparable well on the mini-coco(siou: 0.195, giou 0.198). Since I implement siou before the recent releasement of yolov6, which also applies siou as a loss function, there is a little difference between these two versions. Now I am comparing my paddle version with the torch implementation. Updating will be given later if necessary. As for MobileOne block, I just implement the basic block unit instead of the whole backbone. The way I verify it is to replace this block (with k=1) with the normal convolution block in the head of picodet, which gives an increase of inference speed by 11.4% in CPU mode with comparable mAP performance on my custom dataset.
